src/config.json

```diff
@@ -4,5 +4,12 @@
         "name": "localhost",
         "port": 3000
     },
-    "responderKey": "0x6370fd033278c143179d81c5526140625662b8daa446c22ee2d73db3707e620c"
+    "responderKey": "0x6370fd033278c143179d81c5526140625662b8daa446c22ee2d73db3707e620c",
+    "apiEndpoint": {
```

Comment:
Lets set a default global rate here as well. I think we should also add these options to the command line args, just so that we have parity between them.

Reply:
Done, set 100 request per minute in the default config.json.